biojava-structure/src/main/java/org/biojava/nbio/structure/scop/ScopInstallation.java

```diff
@@ -85,8 +85,8 @@ public class ScopInstallation implements LocalScopDatabase {
 	public static final String comFileName = "dir.com.scop.txt_";
 
 	// Download locations
-	public static final String SCOP_DOWNLOAD = "http://scop.berkeley.edu/downloads/parse/";
-	public static final String SCOP_DOWNLOAD_ALTERNATE = "http://scop.berkeley.edu/downloads/parse/";
```

Comment:
It looks like these changes were from #1077  . I've merged that one now. Could you merge master in to have a cleaner diff?

Reply:
Dang. I thought I was.  This is my first time dealing with a forked repo.  I guess I will use the webapp instead of CLI.  (Sigh)